### DISCUSSION TITLE
Error di Algoritma Leaderboard

### DISCUSSION THREAD

**Discussion by valmayukuk** [teamalgoritma/community#747] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/130435857/b4a546c3-4fbb-4ef9-903c-9243b57dab94)
[valmay_sms-cl-spam.csv](https://github.com/teamalgoritma/community/files/12318103/valmay_sms-cl-spam.csv)

Saya mengalami error ini ketika mengupload data submission. Kira-kira penyebabnya apa? terima kasih. File attachment hasil prediksi sudah saya lampirkan

**Comment by kevbow** [comment]
Dear @valmayukuk,

Jika Anda perhatikan, hasil prediksi model Anda menghasilkan 100% nilai `ham` sehingga Precision bernilai 0. Hal inilah yang menyebabkan error pada leaderboard.
Apabila ingin mencoba, silakan ganti secara manual nilai pada submission Anda cukup 1 baris dengan nilai `spam`.
![image](https://github.com/teamalgoritma/community/assets/68848168/f2cc1308-5763-484f-88d6-8901728e9672)
Berikut saya coba tampilkan contoh yang saya maksud.
Demikian dan semoga membantu. Terima kasih.

**Reply by valmayukuk** [reply]
kak maaf tapi setelah coba upload lagi hasilnya seperti ini
![image](https://github.com/teamalgoritma/community/assets/130435857/6e0e533c-5221-4920-bd03-124f10dc91fc)

**Reply by kevbow** [reply, marked as answer]
Coba dipastikan kembali untuk opsi project yang dipilih sudah diubah ke SMS.
![image](https://github.com/teamalgoritma/community/assets/68848168/b2ea7c5b-7379-48db-a92d-fe417a280867)
